content/scripts/print-diff.js

```diff
@@ -0,0 +1,27 @@
+const chalk = require("chalk");
+
+function printBasicDiff(str1, str2) {
```

Comment:
I whipped this together really quick. It's not perfect nor am I sure it needs to be. 
Things like [`jsdiff`](https://www.npmjs.com/package/diff) can be used in Node but it only gives you a data structure of the points. I.e. at what point does the add/remove happen etc. It doesn't help you printing. You have to do that yourself. 

I'm afraid of perfecting this more than necessary because I don't think it's important enough. Also, perhaps in a future version we might use the Toolbar and the server to trigger a build with `this.options.fixFlaws` set to `true`.

Reply:
Seems to work pretty well as is!